### DISCUSSION TITLE
Duplicate Button disappeared

### DISCUSSION THREAD

**Discussion by geekblog-dev** [dani-garcia/vaultwarden#3468] [Q&A]
### Subject of the issue
Duplication Button disappeared.

### Deployment environment
Debian server with docker

* vaultwarden version:1.28.1

* Install method: Docker image

* Clients used: web desktop ios android

* Reverse proxy and version: nginx 1.22.0

* MySQL/MariaDB or PostgreSQL version: docker

* Other relevant details:

### Steps to reproduce
Update to newest version 

### Expected behaviour
Somewhere in the last versions the option to duplicate the Entrys in the Vault disappeared.

So expect would be to click on any entry and duplicate it to somewhere else.

### Actual behaviour
Button is no longer available.

### Troubleshooting data
Please contact me fore more information because i don't thing any logs are helpful here.

**Comment by BlackDex** [comment]
The web-vault has had several ui updates, depending on your previous version it could be lot.

For me, it all works as intended.
I suggest to read the following pages.

https://bitwarden.com/learning/moving-items-to-your-organization/
https://bitwarden.com/help/managing-items/
